addons/test_mail/tests/test_message_post.py

```diff
@@ -109,7 +111,19 @@ def test_notify_by_mail_add_signature(self):
         self.assertEqual(found_mail.body_html.count(signature), 0)
 
     @users('employee')
-    def test_notify_by_email_prepare_rendering_contextt(self):
+    def test_notify_mail_add_signature_no_author_user_or_no_user(self):
```

Comment:
Would update name to "test_notify_by_mail_add_signature_no_author_user_or_no_user" to match new naming of other tests